.all-contributorsrc

```diff
@@ -1,6 +1,6 @@
 {
   "files": [
-    "README.md"
+    ".all-contributors.md"
```

Comment:
> Have you tried `/dev/null`? 🤔

Reply:
I tried and it doesn't work: `Error: ENOENT: no such file or directory`